src/components/cards/InfoCard.vue

```diff
@@ -39,6 +39,20 @@
             >{{ cardData.subValues.deceased | number }}
           </span>
         </div>
+        <div
+          v-if="cardData.subValues && cardData.subValues.positive"
+          class="card-diff-item"
+        >
+          <div class="trend-icon positive"></div>
+          <span class="positive bad">{{ cardData.subValues.positive | number }}</span>
+        </div>
+        <div
+          v-if="cardData.subValues && cardData.subValues.percent"
+          class="card-diff-item"
+        >
+          <div class="trend-icon percent"></div>
+          <span class="percent tests">{{ cardData.subValues.percent | number }}</span>
```

Comment:
Ne vem v kaksni obliki pride vrednost, ampak mamo tudi filter `percent`, da ti ga formatira po jeziku pravilno in doda znak.

Reply:
aha, a number ne pozna decimalnih mest?
ker percent filter ne bi uporabil, ker doda % na koncu, tukaj pa je ideja da je % že v ikoni

Reply:
Number formatira na 1 mesto po defaultu, sem ti pushal spremembo da sprejme argumente, tko da `number(2, 3)` pomeni min 2 decimalni, max 3 decimalna mesta.

Reply:
ok, čeprav na eno decimalko tudi zadošča.

Reply:
Sej ce pustis kot je bo 1. Torej `| number` bo na 1 decimalko, `| number(2,3)` bo pa na 2-3 decimalke.